docs/modules/ngx_http_dubbo_module.md

```diff
@@ -0,0 +1,279 @@
+ngx_http_dubbo_module
+====
+
+This module provides support for the backend Dubbo support after Tengine version 2.3.2.
+[Apache Dubbo™](http://dubbo.apache.org)  is a high-performance, java based open source RPC framework.It is open source by Alibaba, in years of development, it is one of the most popular microservice framework.
+
+There are Consumer(client) and Provider(Server) in Dubbo. This module support use Tengine as a proxy gateway, front get HTTP/HTTPS/HTTP2 request, and as a Dubbo Consumer pass the request to backend Dubbo Provider service.
```

Comment:
There are two roles, Consumer(client) and Provider(Server) in Dubbo. This module is used to make Tengine as a proxy gateway which receives HTTP/HTTPS/HTTP2 requests at the front then as a Dubbo Consumer passes the requests to backend Dubbo Provider service.